nav2_map_server/src/map_saver_main.cpp

```diff
@@ -24,7 +24,7 @@ const char * USAGE_STRING{
   "Usage: \n"
```

Comment:
We should keep the filename as `_cli` to make it clear there's a "server" and a "cli" option.

Reply:
Fixed: https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-2a1a2abf02d6011dd7803064feb30d95R24, https://github.com/ros-planning/navigation2/commit/84cff7d8cf43443e23bc4852129791d665f6e5c7#diff-2a1a2abf02d6011dd7803064feb30d95R26